features/attach_validtoken.feature

```diff
@@ -98,7 +98,17 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         When I update contract to use `effectiveTo` as `days=-20`
         And I run `python3 /usr/lib/ubuntu-advantage/ua_update_messaging.py` with sudo
         And I run `update-motd` with sudo
-        Then if `<release>` in `xenial or bionic` and stdout matches regexp:
+        Then if `<release>` in `xenial` and stdout matches regexp:
+        """
+
+        \*Your UA Infra: ESM subscription has EXPIRED\*
+
+        \d+ additional security update\(s\) could have been applied via UA Infra: ESM.
+
+        Renew your UA services at https:\/\/ubuntu.com\/16-04
+
+        """
+        Then if `<release>` in `bionic` and stdout matches regexp:
```

Comment:
This test is also failing, but because the "expired" message isn't being printed at all for bionic - not sure if this is in scope of this PR or if you want to break this fix into a different PR